### ISSUE TITLE
Calculator

### ISSUE THREAD

**Issue by rpowis** [hmrc/design-language-documentation#50]
# Screenshot

![image](https://cloud.githubusercontent.com/assets/1752124/22467291/54514594-e7c4-11e6-8137-d6f3fde81cd5.png)

---

![image](https://cloud.githubusercontent.com/assets/1752124/22467312/67ddfc9c-e7c4-11e6-81c0-d74e4be91fc8.png)

**Comment by rpowis** [comment]
Some of these are calculator services rather than a pattern - too big to be a pattern?

**Comment by stevenaproctor** [comment]
@gavinwye I agree with @rpowis that it is too big to be a pattern. It is not one thing on a page. How accessible is this? How easy is it to use on a mobile? Does a calculator need to look like a calculator?

**Comment by gavinwye** [comment]
Comments from London

- We see the need for creating a consistent way of calculating something.
- Could be a pattern, but how? As a journey ( an example of PAYE), all on one page?
- It’s really big, could it be a service? a module?
- Research: what’s the best way of doing it: calculating something through a journey? all on one page to see all the details?
- Does the fact that is a real-time solution or not changes the pattern? Should there be 2 different ways depending on if it’s real time or not?

**Comment by gavinwye** [comment]
We need to define what we mean by calculator and how should calculators be designed 
This is useful and unique 
We should write some guidance for this as a pattern 
There are two types of calculators
- journey based
- play about with some number/variables
this relates to the #3 Spinner UI Control

**Comment by stevenaproctor** [comment]
The issue was closed because there was not enough information about what we mean by a calculator. There was not enough detail about what services use a thing that looks like a calculator and journey-based calculators are difficult to pattern because they are often specific to the service.
